view/frontend/web/js/checkout/shipping_method/show-myparcel-shipping-method.js

```diff
@@ -101,6 +101,7 @@ define(
         function showMailboxRadio() {
             jQuery("td[id^='label_carrier_" + window.mypa.data.general.parent_method + "']").parent().hide();
             jQuery("td[id^='label_carrier_mailbox']").parent().show();
+            jQuery("input[name='delivery_options']").val('{"time":[{"price_comment":"mailbox"}]}');
```

Comment:
Werkt dit ook als je `mailbox met andere verzendmethodes` aan heb staan?

Reply:
Als je `mailbox` aan heb staan met bijvoorbeeld `ophalen in de winkel` en je selecteer `ophalen in de winkel` dan zal er in de quote wel `{"time":[{"price_comment":"mailbox"}]}` te zien zijn. Echter zal er geen mailbox label uitkomen. Omdat er verder in de code gecontroleerd wordt of er gekozen is voor een MyParcel verzendmethode.